src/Makefile.qt.include

```diff
@@ -314,8 +314,7 @@ BITCOIN_QT_CPP += \
   qt/walletview.cpp
 endif
 
-RES_IMAGES = \
-  qt/res/images/about.png
+RES_IMAGES = 
```

Comment:
I'd say keep it empty. If there are images again, they can be added here. There's no need to add another RES_IMAGES to the top of the file.